docs/apache-airflow/core-concepts/dags.rst

```diff
@@ -484,6 +484,125 @@ You can also combine this with the :ref:`concepts:depends-on-past` functionality
     .. image:: /img/branch_with_trigger.png
 
 
+.. _concepts:setup-and-teardown:
+
+Setup and Teardown
+~~~~~~~~~~~~~~~~~~
+
+In data workflows it's common to create resources (such as a compute resource), use it to do some work, and then tear it down. Airflow provides setup and teardown tasks to support this need.
+
+Key features of setup and teardown tasks:
+
+  * If you clear a task, its setups and teardowns will be cleared.
+  * By default, teardown tasks are ignored for the purpose of evaluating dag run state.
+  * A teardown task will run if it's setup was successful, even if its work tasks failed.
+  * Teardown tasks are ignored when setting dependencies against task groups.
+  * A setup task must always have a teardown and vice versa. You may use EmptyOperator as a setup or teardown.
```

Comment:
I don’t think these can be indented? How does this render?

Reply:
Looks fine:
<img width="952" alt="image" src="https://github.com/apache/airflow/assets/15932138/b6cd0cb8-1299-4f5a-8325-4717e0599867">